bundle/jsky.app.ot/src/main/resources/resources/conf/welcome.html

```diff
@@ -30,7 +30,7 @@
 
 <p>The Observing Tool is the work of the Gemini Observatory staff including Bryan Miller, Andrew Stephens, Mischa Schirmer, and the software engineers in the Gemini Engineering Operations Department and former employees Florian Nussberger, Devin Dawson, Larry O'Brien, Nicolas Barriga, Alan Brighton, Vasudeva Upadhya, Lorraine Callahan, Darrell Denlinger, Kim Gillies, and Phil Puxley.</p>
 
-<p>The image at left is a long exposure of Gemini North during Altair laser guide star operations. Credit: Gemini Observatory.</p>
+<p>The image at left shows the interacting galaxy pair NGC 1351 and NGC 1352 imaged with the Gemini Multi-Object Spectrograph (GMOS) on the Gemini South telescope. The larger NGC 1352 is slightly more than the Milky Way while the smaller NGC 1351 is comparable to the Large Magellanic Cloud. Credit: Gemini Observatory.</p>
```

Comment:
"is slightly more"?

Reply:
I thought that sounded awkward and incorrect, too, so I made a comment on the JIRA task.
I'll wait to merge this until I hear back.

Reply:
Right - @sraaphorst asked for clarification in the REL task, so hopefully @andrewwstephens or Bryan can provide the right wording for that.